iac/instance.py

```diff
@@ -42,12 +45,16 @@ def from_str(kind: str) -> InstanceKindSelf:
 
 
 @dataclass(frozen=True)
+# pylint: disable = too-many-instance-attributes
 class Instance:
     name: str
+    state: str
     id: str = None
     key_name: str = None
     tags: [iac.aws.Tag] = None
     public_dns_name: str = None
+    public_ip_address: str = None
+    nitro: bool = None
```

Comment:
maybe out of scope, but it is useful when working with the tool so I added while I was in here.